### PR TITLE
ISL update for new TH1 tunnel

### PR DESCRIPTION
**Pull request by Duke87ofST**

### CODE REVIEW

**Comment by michihdeu** [comment]
@Duke87ofST 

2015-08-25;Italy;A11 (Viareggio);ita.a011via;New Routehttps://github.com/TravelMapping/UserData/pull/2314

File: updates.csv [hunk 1]
```diff
@@ -787,7 +787,7 @@
 2015-12-23;Italy;E35;ita.e35;Removed from A1 and relocated onto A1Var between northern and southern A1/A1Var junctions
 2015-12-06;Italy;A1 Var (Variente di Valico);ita.a001varval;New Route
 2015-08-25;Italy;A1 (Sasso Marconi);ita.a001sas;New Route
-2015-08-25;Italy;A11 (Viareggio);ita.a011via;New Route
+2015-08-25;Italy;A11 (Viareggio);ita.a011via;New Routehttps://github.com/TravelMapping/UserData/pull/2314
```
Comment (jteresco) [inline updates.csv]: I'll remove the extraneous URL that got pasted in after I merge.

Comment (Duke87ofST) [reply updates.csv]: THAT is really strange. I was not editing the file anywhere near that line, nor had I viewed the page at that URL or had any reason to have copied the text.

Comment (yakra) [reply updates.csv:790]: @Duke87ofST, do you use Linux? Looks like the kind of thing that can happen by inadvertently mashing the middle mouse button.

Comment (Duke87ofST) [reply updates.csv:790]: I do, yes - Ubuntu specifically. Middle mouse button eh? Guess I'll have to watch out for that.

**Pull request merged**